packages/mask/src/plugins/Collectible/SNSAdaptor/MakeOfferDialog.tsx

```diff
@@ -111,7 +112,8 @@ export function MakeOfferDialog(props: MakeOfferDialogProps) {
             expirationTime: !isAuction ? getUnixTime(expirationDateTime) : undefined,
             paymentTokenAddress: token.value.type === EthereumTokenType.Native ? undefined : token.value.address,
         })
-    }, [asset?.value, token, account, amount, expirationDateTime, isAuction])
+        activatedSocialNetworkUI.utils.share?.(t('promote_collectible'))
+    }, [asset?.value, token, account, amount, expirationDateTime, isAuction, t])
```

Comment:
```suggestion
    }, [asset?.value, token, account, amount, expirationDateTime, isAuction])
```

Reply:
`t` is used. `t('promote_collectible')`

Reply:
IIRC `t` will change if the language changes.